javascript/ql/src/semmle/javascript/StandardLibrary.qll

```diff
@@ -146,3 +146,37 @@ class StringReplaceCall extends DataFlow::MethodCallNode {
     )
   }
 }
+
+/**
+ * A call to `String.prototype.split`.
+ *
+ * We heuristically include any call to a method called `split`, provided it either
+ * has exactly one arguments, or local data flow suggests that the receiver may be a string.
+ */
+class StringSplitCall extends DataFlow::MethodCallNode {
+  StringSplitCall() {
+    this.getMethodName() = "split" and
+    (getNumArgument() = 1 or getReceiver().mayHaveStringValue(_))
```

Comment:
The signature is: `str.split([separator[, limit]])`
(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split)